src/main/java/edu/kit/datamanager/metastore2/web/impl/SchemaRegistryControllerUIImpl.java

```diff
@@ -56,59 +52,80 @@
     private final static String UIFORMSCHEMA = "/static/jsonSchemas/uiFormSchemaRecord.json";
     private final static String ITEMSSCHEMA = "/static/jsonSchemas/itemsSchemaRecord.json";
     
-            private final static String DATAMODELMETADATA = "/static/jsonSchemas/metadataRecord.json";
+    private final static String DATAMODELMETADATA = "/static/jsonSchemas/metadataRecord.json";
     private final static String UIFORMMETADATA = "/static/jsonSchemas/uiFormMetadataRecord.json";
     private final static String ITEMSMETADATA = "/static/jsonSchemas/itemsMetadataRecord.json";
-
+    
```

Comment:
Could you please move all REST-endpoints regarding UI to a separate java class!? 
Furthermore, I would appreciate it if all the code concerning metadata editor would end up in a separate library. But we can address this later.

Reply:
One more thing: 
Try to follow the Java conventions and make it more flexible(?): 
UIFORMMETADATA is hard to read use instead: UI_FORM_METADATA
String PATH_TO_STATIC_DOCUMENTS = "/static"
String  PATH_TO_JSON_SCHEMAS =  PATH_TO_STATIC_DOCUMENTS + "/jsonSchemas"
String UI_FORM_METADATA = PATH_TO_JSON_SCHEMAS + "/uiFormMetadataRecord.json"

Reply:
> Could you please move all REST-endpoints regarding UI to a separate java class!? Furthermore, I would appreciate it if all the code concerning metadata editor would end up in a separate library. But we can address this later.

It's already in a separate Class for the editor UI